nimbus-features/messagingFeature.yaml

```diff
@@ -80,6 +80,7 @@ features:
           # Nimbus team.
           triggers:
             NOT_INSTALLED_TODAY:      days_since_install > 0
+            DAY_3_AFTER_INSTALL:      days_since_install == 3
```

Comment:
As we cannot predict when iOS will launch the app for background refresh and it might be on day 4 I would suggest to change it `>=` (so `days_since_install >= 3`).